src/voigt.jl

```diff
@@ -193,3 +202,10 @@ end
 Base.@propagate_inbounds function frommandel(TT::Type{<: SymmetricTensor{4,dim}}, v::AbstractMatrix{T}; offset_i::Int=0, offset_j::Int=0, order=DEFAULT_VOIGT_ORDER[dim]) where {T,dim}
     fromvoigt(TT, v, offdiagscale=T(√2), offset_i=offset_i, offset_j=offset_j, order=order)
 end
+
+Base.@propagate_inbounds function frommandel(TT::Type{<: Tensor{2,dim}}, v::AbstractVector{T}; offset::Int=0, order=DEFAULT_VOIGT_ORDER[dim]) where {T,dim}
+    fromvoigt(TT, v, offset=offset, order=order)
+end
+Base.@propagate_inbounds function frommandel(TT::Type{<: Tensor{4,dim}}, v::AbstractMatrix{T}; offset_i::Int=0, offset_j::Int=0, order=DEFAULT_VOIGT_ORDER[dim]) where {T,dim}
+    fromvoigt(TT, v, offset_i=offset_i, offset_j=offset_j, order=order)
+end
```

Comment:
And here
```suggestion
Base.@propagate_inbounds frommandel(TT::Type{<: Tensor{2}}, v::AbstractVector{T}; kwargs...) = fromvoigt(TT, v; kwargs...)
Base.@propagate_inbounds frommandel(TT::Type{<: Tensor{4}}, v::AbstractMatrix{T}; kwargs...) = fromvoigt(TT, v; kwargs...)
```

Reply:
Should this also be extended to `Tensors{1}` then?

Reply:
Yea, why not, seems useful